HPXMLtoOpenStudio/measure.rb

```diff
@@ -4169,15 +4191,102 @@ def self.set_surface_exterior(model, spaces, surface, exterior_adjacent_to)
     elsif [HPXML::LocationBasementConditioned].include? exterior_adjacent_to
       surface.createAdjacentSurface(create_or_get_space(model, spaces, HPXML::LocationLivingSpace))
       @cond_bsmnt_surfaces << surface
+    elsif [HPXML::LocationOtherHeatedSpace, HPXML::LocationOtherMultifamilyBufferSpace, HPXML::LocationOtherNonFreezingSpace].include? exterior_adjacent_to
+      add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
+      surface.setSunExposure('NoSun')
+      surface.setWindExposure('NoWind')
     else
       surface.createAdjacentSurface(create_or_get_space(model, spaces, exterior_adjacent_to))
     end
   end
 
+  def self.add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
+    if spaces[exterior_adjacent_to].nil?
+      # Create E+ other side coefficient object
+      otherside_object = OpenStudio::Model::SurfacePropertyOtherSideCoefficients.new(model)
+      otherside_object.setName(exterior_adjacent_to)
+      # Fixme: assumption the same as SurfacePropertyConvectionCoefficients of return air plenum
+      otherside_object.setCombinedConvectiveRadiativeFilmCoefficient(30)
```

Comment:
Hmm. I'm inclined to say we should just use a value that is <= 0. See https://bigladdersoftware.com/epx/docs/9-2/input-output-reference/group-advanced-surface-concepts.html#field-combined-convectiveradiative-film-coefficient

Reply:
In that way, the surface exterior temperature will be exactly the new space temperature calculated: instead of assuming there's air between.

Reply:
I realize that. It just seemed like an okay assumption given that the temperatures are already completely arbitrary?